src/components/layout/Navbar/index.tsx

```diff
@@ -104,6 +107,14 @@ const Navbar = ({ user }: NavbarProps) => {
           className={styles.mobileNavItem}
           onClick={() => setMenuOpen(false)}
           href={config.homeRoute}
+        >
+          <DashboardIcon className={styles.iconLink} />
+          Dashboard
+        </Link>
+        <Link
+          className={styles.mobileNavItem}
+          onClick={() => setMenuOpen(false)}
```

Comment:
We could probably refactor this part of the navbar by taking advantage of event propagation and moving the repeated `onClick={() => setMenuOpen(false)}` to the parent `<nav>`. If we only want the mobile navbar to close when the user presses a link, you could add a check for that using [Element.closest](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)

```tsx
<nav className={styles.portalLinks} onClick={e => e.target.closest('a') && setMenuOpen(false)}>
```

I don't think this will type check, though, since using `event.target` in TypeScript is kind of hairy. might have to do something like `e.target instanceof Element && ...`

Reply:
done! I disabled an ESLint rule here since they were complaining about `onClick`s for a div and I didn't want to get too deep into the weeds on that lol